### DISCUSSION TITLE
Captain's Log - Separate App?

### DISCUSSION THREAD

**Discussion by TakeshiYamato01** [bcholmes/StarTrek2d20#162] [Ideas]
So, given what I've read in the Captain's Log PDF, I'm not sure that it's 100% compatible with the current app as is. About the main thing that would be useful for mainline STA games would be the Matrices in the back of the book, for encounters, mission types, NPCs and the like - and especially the Yes/No Probability Matrix.

Would it be possible to split off Captain's Log into its own App? You wouldn't have to implement Focuses or Personal Talents into a Captain's Log App, and with ships they all seem to have a Mission Profile pre-assigned when it comes to Department Values. The biggest hurdle I can see would be figuring out what Mission Profiles would be most common for KDF starship classes in that regard.

**Comment by Felderburg** [comment]
I haven't read Captain's Log, but aren't the matrices sort of... the whole thing? It's my understanding that you use the tables to build the actual missions, which makes it seem like putting them in a free tool might infringe a bit too much on Modiphius' sales of the CL book.

**Reply by TakeshiYamato01** [reply]
I'm not saying putting all the matrices in an app - I'm just saying the character creation process in Captain's Log is ***just*** different enough from baseline STA to be incompatible with the app in its current form, and a separate app for Captain's Log is the solution that makes the most sense.

**Reply by Felderburg** [reply]
Ok, I misunderstood because you said "the main thing that would be useful for mainline STA games would be the Matrices" which I took to mean you wanted them in a separate app.

@bcholmes has already taken steps to incorporate CL into this tool. I'm not sure what that entails, but it seems that the intention is for this to remain the one-stop-shop for all charcater creation.

**Reply by TakeshiYamato01** [reply]
Fair enough.
